### PR TITLE
Add ImpeREAL Empire Unique Cities - Sutch

### PR DESCRIPTION
**Pull request by charabaruk**
Includes #1623

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -34013,3 +34020,16 @@ plugins:
...
+      - link: 'https://www.nexusmods.com/oblivion/mods/37867/'
+        name: 'ImpeREAL Empire Unique Cities - Sutch'
+    dirty:
+      - <<: *quickClean
+        crc: 0xEB59AB31
+        util: '[TES4Edit v4.0.4](https://www.nexusmods.com/oblivion/mods/11536)'
+        itm: 2
+    clean:
+      - crc: 0x0872D0D1
+        util: 'TES4Edit v4.0.4'
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
```

Explained in #1621.

**Review by sibir-ine — CHANGES_REQUESTED** [review]

File: masterlist.yaml (hunk 1 above)
Comment (coldacid) [reply masterlist.yaml:34035]: This will come out after rebasing e7d7d9b917d82bbdda0448d8c3af81a9e48f3f67 on the final commit for #1621

**Comment by charabaruk** [comment]
This should be good once #1623 is merged

**Comment by charabaruk** [comment]
@sibir-ine it looks like GitHub thinks some of the already-merged changes from #1623 are getting merged again in this; want me to rebase? Same thing is happening with the other two remaining PRs as well.

**Comment by sibir-ine** [comment]
Yes please, some changes were made since your last rebase (i.e., the location & plugin name), so merging now wouldn't be clean.

**Comment by charabaruk** [comment]
All three branches are now rebased.

**Review by sibir-ine — APPROVED** [review]

**Pull request merged**